parachain/primitives/core/src/lib.rs

```diff
@@ -31,9 +31,27 @@ pub trait Verifier<AccountId> {
 	fn verify_bulk(sender: AccountId, messages: &[(AppId, Message)]) -> DispatchResult;
 }
 
-/// A trait for handling message payloads.
-///
-/// This trait should be implemented by runtime modules that wish to handle message payloads.
+impl<AccountId> Verifier<AccountId> for () {
+	fn verify(sender: AccountId, app_id: AppId, message: &Message) -> DispatchResult {
+		Ok(())
+	}
+	fn verify_bulk(sender: AccountId, messages: &[(AppId, Message)]) -> DispatchResult {
+		Ok(())
+	}
+}
+
+/// Outbound submission for applications
+pub trait SubmitOutbound {
+	fn submit(channel_id: ChannelId, payload: &[u8]) -> DispatchResult;
+}
+
+impl SubmitOutbound for () {
+	fn submit(channel_id: ChannelId, payload: &[u8]) -> DispatchResult {
+		Ok(())
+	}
+}
```

Comment:
Could you add a comment here why we have a default implementation that does not err?